masonry/src/widget/zstack.rs

```diff
@@ -0,0 +1,324 @@
+// Copyright 2024 the Xilem Authors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    vello::Scene, widget::WidgetMut, AccessCtx, BoxConstraints, LayoutCtx, PaintCtx, Point,
+    QueryCtx, RegisterCtx, Size, Widget, WidgetId, WidgetPod,
+};
+use accesskit::{NodeBuilder, Role};
+use smallvec::SmallVec;
+use tracing::trace_span;
+
+struct Child {
+    widget: WidgetPod<Box<dyn Widget>>,
```

Comment:
Is there a reason the children don't have an individual alignment?

Reply:
Yeah, this would be the right way to expose that at the Xilem level.

Reply:
Should we implement this or keep it simple like it is now?

Reply:
I think we can implement it, I don't think it will be much more complex? It's a little bit more code, but it would be consistent to the other container views, and may be more straight-forward on the user-facing-side.
I think it's a little bit awkward to compose 2 `zstack` to achieve similar behavior.
(And the wrapper types (`Child`, `ZStackElement`) would actually make sense).

Reply:
Added in bbb1d5f6bcbc1c5e1fc40cf7ee54b781caad2550